packages/terra-cell-grid/src/terra-dev-site/doc/cell-grid/CellGrid.1.doc.jsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
```

Comment:
Great candidate for mxd and the new loaders 😄

Reply:
these files look so ugly compared to the mdx 😄